env.py

```diff
@@ -155,6 +157,15 @@ def get_env_vars(
             "Remove users no longer in this organization from CODEOWNERS file"
         )
 
+    issue_report = os.getenv("ISSUE_REPORT")
```

Comment:
Do we want to reuse [this helper function](https://github.com/github/evergreen/blob/b52dc5889ba82ea6bae2e30b11b3570921f01994/env.py#L12-L25) in evergreen here or in another PR?

Reply:
Lets do that here. I think its related enough since we are adding duplicate code.

Reply:
On it.  I'll push commit

Reply:
This applied to `ISSUE_REPORT` and `DRY_RUN` env vars